core/src/main/java/ai/timefold/solver/core/impl/move/streams/maybeapi/stream/MoveStreamFactory.java

```diff
@@ -35,33 +35,33 @@ public interface MoveStreamFactory<Solution_> {
      * @return A stream containing a tuple for each of the entities as described above.
      * @see PlanningPin An annotation to mark the entire entity as pinned.
      * @see PlanningPinToIndex An annotation to specify only a portion of {@link PlanningListVariable} is pinned.
-     * @see #enumerateIncludingPinned(Class, boolean) Specialized method exists to automatically include pinned entities as
+     * @see #forEachUnfiltered(Class, boolean) Specialized method exists to automatically include pinned entities as
      *      well.
      */
-    <A> UniDataStream<Solution_, A> enumerate(Class<A> sourceClass, boolean includeNull);
+    <A> UniDataStream<Solution_, A> forEach(Class<A> sourceClass, boolean includeNull);
 
     /**
      * Start a {@link DataStream} of all instances of the sourceClass
      * that are known as {@link ProblemFactCollectionProperty problem facts}
      * or {@link PlanningEntity planning entities}.
      * If the sourceClass is a genuine or shadow entity,
      * it returns instances regardless of their pinning status.
-     * Otherwise as defined by {@link #enumerate(Class, boolean)}.
+     * Otherwise as defined by {@link #forEach(Class, boolean)}.
      */
-    <A> UniDataStream<Solution_, A> enumerateIncludingPinned(Class<A> sourceClass, boolean includeNull);
+    <A> UniDataStream<Solution_, A> forEachUnfiltered(Class<A> sourceClass, boolean includeNull);
```

Comment:
I wonder if we should keep the term "IncludingPinned." It would make sense without needing to read the documentation.